scriptmodules/supplementary/launchingimages.sh

```diff
@@ -0,0 +1,413 @@
+#!/usr/bin/env bash
+
+# This file is part of The RetroPie Project
+#
+# The RetroPie Project is the legal property of its developers, whose names are
+# too numerous to list here. Please refer to the COPYRIGHT.md file distributed with this source.
+#
+# See the LICENSE.md file at the top-level directory of this distribution and
+# at https://raw.githubusercontent.com/RetroPie/RetroPie-Setup/master/LICENSE.md
+#
+
+rp_module_id="launchingimages"
+rp_module_desc="Generate runcommand launching images based on emulationstation themes."
+rp_module_help="A runcommand launching image is displayed while loading a game, with this tool you can automatically create some cool images based on a chosen emulationstation theme you have on your system."
+rp_module_section="exp"
+rp_module_flags="noinstclean"
+
+function depends_launchingimages() {
+    local depends=(imagemagick librsvg2-bin)
+    if isPlatform "x11"; then
+        getDepends feh
+    else
+        getDepends fbi
+    fi
+    getDepends "${depends[@]}"
+}
+
+function sources_launchingimages() {
+    gitPullOrClone "$md_inst" "https://github.com/meleu/generate-launching-images.git"
+}
+
+function install_launchingimages() {
+    cd "$md_inst"
+    chmod a+x generate-launching-images.sh
```

Comment:
the file permissions could also be committed to git, which would remove the need for the chmod